src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -82,11 +82,19 @@ void ResolveItemMonikers(List<TableOfContentsItem> items)
                     }
 
                     List<string> monikers = null;
+
+                    var linkType = UrlUtility.GetLinkType(item.Href?.Value);
+                    if (linkType == LinkType.External || linkType == LinkType.AbsolutePath)
```

Comment:
Instead of checking for link type, is it easier to detect pure text node?

Reply:
there are three cases need to check

Reply:
can we reuse [TocHrefType](https://github.com/dotnet/docfx/blob/b67e013094b9a7b868f60d86133a7e27943cd3c7/src/docfx/build/toc/parser/TableOfContentsParser.cs#L350) here?

Reply:
that require saving that info in the tocModel